SubterfugeCore/Core/Players/Player.cs

```diff
@@ -84,5 +116,27 @@ public string GetPlayerName()
         {
             return this.PlayerName;
         }
+
+        /// <summary>
+        /// Get the player's driller capacity
+        /// </summary>
+        /// <returns>The player's driller capacity</returns>
+        public int GetDrillerCapacity()
+        {
+            List<Outpost> playerOutposts = Game.TimeMachine.GetState().GetOutposts();
+            int capacity = 0; // base capacity?
+
+            // Generators increase capacity
+            foreach (Outpost outpost in playerOutposts)
+            {
+                if(outpost.GetOutpostType().Equals(OutpostType.Generator))
+                {
+                    capacity += 50;
+                }
+
+            }
+            this.drillerCapacity = capacity;
```

Comment:
Here we set a private variable and then return it, but the only way to access this variable is by calling this method. I would just remove the variable and return the capacity, I don't think we need to store the drillerCapacity as a variable.

This also seems to be the cause of an error in the build pipeline:
```Core/Players/Player.cs(138,13): error CS0200: Property or indexer 'Player.drillerCapacity' cannot be assigned to -- it is read only [/__w/Remake-Core/Remake-Core/SubterfugeCore/SubterfugeCore.csproj]```

Reply:
I'm kinda unhappy that the getter-method calculates the capacity. Same for the driller count.
I thought about maybe making the driller cap calculation another method and the getter will just return the variable. Then maybe only call the calculation method when the capacity changes; to update it.
Would that save some performance? Or is it not worth the hassle?

Reply:
> Would that save some performance?
Honestly, using the for loop to determine capacity and driller count is quite fast performance wise. The only improvement would be to store the total driller count and update it each time the player loses/gains drillers so that it is always up to date. That might be better but in terms of performance, it really doesn't make a significant difference.

Reply:
Alright, then I'll leave it that way and will remove the variable for now.